### PR TITLE
Make update

### PR DESCRIPTION
**Pull request by travisle22**
Just something that make testing easier?
and improvements to code

### CODE REVIEW

File: project-2/Makefile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 CC = g++
 
-edit : main.o linked.o hpfnp.o
-	g++ -o test main.o linked.o hpfnp.o
+edit : main.o linked.o hpfnp.o roundRobin.o FCFS.o stats.o
+	g++ -o test main.o linked.o hpfnp.o roundRobin.o FCFS.o \
```
Comment (qabach) [inline project-2/Makefile]: sorry I forgot to ask earlier. Could you add the following s.t. g++ will compile in C++11 standard: 

g++ -std=c++11 ...

Thank you, Travis. 

Quan

**Review by qabach — APPROVED** [review]
looks good

**Review by anhtruong18 — APPROVED** [review]
look fine!

**Comment by qabach** [comment]
got 2 approved. Merge in.

**Pull request merged**